docs/pages/connect-your-client/vnet.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Using VNet
+description: Using VNet
+---
+
+## How it works
+
+VNet is a virtual network emulation tool which enables any program on your computer to connect to
+TCP applications protected by Teleport, with no extra changes required to the program itself. It is
+available in Teleport Connect and tsh on macOS, with Windows and Linux version in the works.
+
+Let's say you work on software which needs to connect to a TCP-based application accessible
```

Comment:
I would try to explain the benefit of VNet without talking about an earlier version of Teleport, since a reader may not be familiar with the earlier version.